env0/utils.go

```diff
@@ -327,6 +368,17 @@ func getResourceDataSliceStructValue(val reflect.Value, name string, d *schema.R
 			continue
 		}
 
+		if fieldType.Kind() == reflect.Slice {
```

Comment:
new code to handle more use-cases (see the new unit test).
Basically, list in list with struct... (yeah, sounds crazy...).

Reply:
Which unit test ?

Reply:
sorry... had to remove the unit tests.
I don't have a real use case for it at the moment. Therefore, had to remove the unit test.
With that said it could become useful in the future so I left the code.